src/profile-logic/tracks.js

```diff
@@ -555,6 +562,12 @@ function getDefaultSelectedThreadIndexes(
   visibleThreadIndexes: ThreadIndex[],
   profile: Profile
 ): Set<ThreadIndex> {
+  if (profile.meta.initialSelectedThreads !== undefined) {
+    profile.meta.initialSelectedThreads.forEach((index) =>
+      ensureExists(profile.threads[index])
```

Comment:
If you really want to crash, I'd suggest to do something such as:
```js
if (index >= profile.threads.length) {
  throw new Error(`The specified thread index ${index} is higher than the maximum thread index ${profile.threads.length - 1}.`);
}
```
Otherwise I recommend that we do a filter operation instead:
```js
return new Set(profile.meta.initialSelectedThreads.filter(threadIndex => {
  if (index < profile.threads.length) {
    return true;
  }

  console.warn(`The specified thread index ${index} is higher than the maximum thread index ${profile.threads.length - 1}.`)
  return false;
});
```

Reply:
I used the pattern seen before.